drivers/spi/Kconfig.dw

```diff
@@ -6,8 +6,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# This option signifies whether DesignWare SPI compatible HW is
+# available
+config HAS_SPI_DW
+	bool
```

Comment:
More of a general comment, but help texts are usually better than comments even for promptless helper symbols, as the `menuconfig` supports viewing the help text of invisible symbols as well (in show-all mode, which gets turned on automatically if you jump to an invisible symbol).